src/ar_fork_recovery.erl

```diff
@@ -302,7 +302,7 @@ do_fork_recover(S = #state {
 					)
 				of
 					false ->
-						ar:err(
+						ar:report_console(
```

Comment:
Why not `ar:info` (here and in the following one)?

Reply:
I'm thinking it might make sense to do a `ar:console/1` call here, but since `ar:console/1 doesn't format this message nicely, it's better to leave it as `ar:report_console/1` so we can deal with it later.

If you don't think it makes sense to log it to the console, then we can change it to `ar:info/1`.

Reply:
I changed to `ar:info` for "the following one" (applied_fork_recovery_block)

Reply:
> If you don't think it makes sense to log it to the console

It's not logged to the console anyway, because of a macro, unless I am missing smth. Under the hood `ar:info` would do the same thing as `report_console`.